### DISCUSSION TITLE
[UX] - Make resolution suggestions without closing the mediation ticket

### DISCUSSION THREAD

**Discussion by huey735** [bisq-network/bisq#5225] [Ideas]
At present, the resolution suggestion and and the closing of a ticket are near synonymous. It makes to leave the closure of the ticket separated so that the traders and mediator can continue to talk.

@jmacxx @leo816

**Comment by ghost** [comment]
Sounds good, I'll add it to the TODO list in https://github.com/bisq-network/bisq/pull/5207
